pages/courseSyncs/courseSyncs.js

```diff
@@ -45,11 +46,20 @@ var pullAndUpdate = function(locals, callback) {
         // continue executing below to launch the jobs themselves.
 
         // First define the jobs.
+        //
+        // After either cloning or pulling from Git, we'll need to load and
+        // store the current commit hash in the database
+        const updateCommitHash = () => {
+            courseUtil.updateCourseCommitHash(locals.course, (err) => {
+                ERR(err, (e) => logger.error(e));
+                syncStage2();
+            });
+        };
```

Comment:
I was keeping these blocks in the execution order in the code and numbering them sequentially, so we run 1A->2->3 (or 1B->2->3).

Reply:
I did this so that this wouldn't become a separate stage, that feels like unnecessary clutter in the frontend. I can make that change though if you want.

Reply:
ah... I misread the control flow and didn't notice that updateCommitHash() calls syncStage2() without spawning another job to do so. All clear now!

But thinking further, how about moving this inside syncFromDisk.syncDiskToSql()? Seems like it's naturally part of syncing everything from disk into the SQL DB.